langs/jbang.yaml

```diff
@@ -0,0 +1,138 @@
+id: "jbang"
+name: "Java w/JBang"
```

Comment:
btw. not too hapy about this name but just putting "JBang" means java users wont find it.